src/utilities/effects.ts

```diff
@@ -4826,4 +4864,40 @@ function replacePathsHelper(schema: ValueSchema, arg: Argument, pathsToReplace:
   }
 }
 
+// Sort events using their dense time markings
+function compareEvents(a: RawSimulationEvent, b: RawSimulationEvent) {
```

Comment:
Could you move this to the `simulation.ts` utils file (or another utils file if you think it is better suited to another one?). Makes it easier to write a test which might also be a good thing to add here?

Reply:
Great idea - I've gone ahead and added a `simulation.test.ts`. Let me know if you think that's the right place for this kind of test

Reply:
Cool and yeah that'd be the right place for it. Not actually seeing the test in there though - did it make it into the last commit?

Reply:
You're right - it didn't make it into the commit 🤦 Should be there now